src/examples/libpmemobj/libart/libart1/Makefile

```diff
@@ -0,0 +1,73 @@
+#
+# Copyright 2016, FUJITSU TECHNOLOGY SOLUTIONS GMBH
+# Copyright 2016, Intel Corporation
```

Comment:
Oh, I see, the commit wasn't made by either Intel or Fujitsu.
Nevermind then! The license checking script works perfectly.